styles.css

```diff
@@ -60,21 +64,221 @@ main {
   padding-top: 12px;
 }
 
-.page1 h2 {
+.sub-txt {
   font-weight: 500;
   font-size: 16px;
   line-height: 24px;
   color: #7f8cff;
   padding-top: 12px;
 }
 
-.page1 ul {
+.social {
   display: flex;
   padding-top: 12px;
-  padding-bottom: 136px;
   list-style: none;
 }
 
-.page1 ul li {
+.social li {
   margin-right: 20px;
 }
+
+.works {
+  width: 100%;
+  margin-top: 114px;
+  padding: 0 16px;
+}
```

Comment:
- [ ] As the project required, it is mandatory to use flexbox and grid. For the main container of the works section, you should use css grid. Kindly use it to match the project requirements.

Reply:
@leonardodiasb, I insert `css grid` in the works section.
![image](https://user-images.githubusercontent.com/110146573/221319123-af30883f-c9b7-4579-94d8-c90fd3ea759b.png)